### ISSUE TITLE
File browser tabs

### ISSUE THREAD

**Issue by dgelessus** [omz/Pythonista-Issues#29]
The file browser should have some kind of tabbing support. At the moment it's kind of annoying to work in multiple directories at once, because you need to "walk" the entire way between them by hand. Having tabs in the file browser "sidebar" might not be that useful because there's so little space, so maybe there should be a way to open a folder in a normal editor tab.

**Comment by danrcook** [comment]
I agree. My thought though is just even a kind of foldable folder tree view would work well. Something similar to when creating a new file.

**Comment by dgelessus** [comment]
That would work too. (But the sidebar would probably need to be wider for that to work well, otherwise you'd run out of space after 5 levels of folders or so.)

**Comment by danrcook** [comment]
Another style: project views (stacked, foldable) that can be easily accessible. Code editors like ATOM have a project view, but having multiple project folders available would be great.

A project view would include its own files and folders.
Example: RSS Reader

**RSS Reader** *(a folder in the filesystem....containing:)*
+---Some folder
      +---some file
+---rssreader.py
+---rssreader.pyui

And repeat for other projects

**Rough idea:**
![image](https://cloud.githubusercontent.com/assets/12954427/16170624/33bbb680-3594-11e6-9c32-ae689a3842e7.png)
